store/src/db/sql.rs

```diff
@@ -113,6 +115,33 @@ pub fn select_accounts_by_block_range(
     Ok(result)
 }
 
+/// Select the latest account details by account id from the DB using the given [Connection].
+///
+/// # Returns
+///
+/// The latest account details, or an error.
+pub fn get_account_details(
```

Comment:
Nit: The functions that do a single `SELECT` are named with `select_`. Maybe lets rename it so that it is consistent?

(the `get_state_sync` is named so because it does much more work that a single `SELECT`)